core/badge-urls/make-badge-url.js

```diff
@@ -61,8 +61,8 @@ function staticBadgeUrl({
   style,
   format = 'svg',
 }) {
-  if (!label || !message) {
-    throw Error('label and message are required')
+  if (!message) {
```

Comment:
What are the implications of `label` no longer being required?

Reply:
It lets you use this function to make a badge without a label, nothing beyond that.

Reply:
👍